src/V3Randomize.cpp

```diff
@@ -155,8 +155,8 @@ class RandomizeMarkVisitor final : public VNVisitor {
                     }
                     // If the class is randomized inline, all members use rand mode
                     if (nodep->user1() == IS_RANDOMIZED_INLINE) {
-                        VarRandMode randMode = {};
-                        randMode.usesRandMode = true;
+                        RandomizeMode randMode = {};
```

Comment:
I realize we had this before, but looking it up it I think union `= {}` was not legal in C++14 (https://en.cppreference.com/w/c/language/struct_initialization).  Instead make RandomizeMode a class with a single union member, and use the class constructor to zero asInt.  Also as part of that rename members to m_usesRandMode, m_index, m_asInt.

Reply:
The page you linked is for C. I think [it's been legal in C++ since before C++11]( https://en.cppreference.com/w/cpp/language/aggregate_initialization). The one thing to keep in mind is it'll only initialize the first member (so its size should be >= than the other members, as it is in this case).